soc/xtensa/intel_adsp/ace_v1x/power.c

```diff
@@ -5,16 +5,30 @@
  */
 #include <zephyr/zephyr.h>
 #include <zephyr/pm/pm.h>
+#include <cpu_init.h>
 
 #include <ace_v1x-regs.h>
 
+#define CHECK_DELAY             256
+#define LPSRAM_MAGIC_VALUE      0x13579BDF
+#define LPSCTL_BATTR_MASK       BIT_MASK_FROM_RANGE(12, 16)
 #define SRAM_ALIAS_BASE         0xA0000000
 #define SRAM_ALIAS_MASK         0xF0000000
+#define MEMCTL_INIT_BIT         23
```

Comment:
Why not `#define MEMCTL_INIT_BIT        BIT(23)` ?
And then in the next line you just use the macro ?